tests/system/load/test_large_tables.py

```diff
@@ -74,13 +74,9 @@ def test_index_repr_large_table():
     assert actual is not None
 
 
-# FAILED
-# tests/system/load/test_large_tables.py::test_to_pandas_batches_large_table
-# google.api_core.exceptions.Forbidden: 403 Response too large to return.
-# Consider specifying a destination table in your job...
-@pytest.mark.xfail
 def test_to_pandas_batches_large_table():
-    df = bpd.read_gbq("load_testing.scalars_100gb")
+    df = bpd.read_gbq("load_testing.scalars_1gb")
```

Comment:
can we take a middle value, say 20gb to still represent a large table. I suspect even pandas may be able to handle 1gb.

Reply:
I think a middle value would probably work, but I don't think we're really testing hardly anything here other than the download speed of the local machine? Even if it was an error in read_gbq, we wouldn't catch that. The middle layer is the BigQuery python library and then it's Bigframes. I don't quite see what load-related errors we're catching? 1 GB is enough to test the round trip... more seems like just testing downloading, what do you think?

Reply:
my understanding is that it's really the load - more the better. I agree it depends on the size of the VM running the test, but we should test as large as we can.

Reply:
Hm, alright, well the available middle ground is 10GB, so I'll switch to that. We also have other tests that do 1TB and such in this file, so we should be all set with this change, I think.